domain/src/main/java/be/xplore/githubmetrics/domain/providers/usecases/DefaultGetAllJobsOfLastDayUseCase.java

```diff
@@ -0,0 +1,28 @@
+package be.xplore.githubmetrics.domain.providers.usecases;
+
+import be.xplore.githubmetrics.domain.domain.Job;
+import be.xplore.githubmetrics.domain.domain.WorkflowRun;
+import be.xplore.githubmetrics.domain.providers.ports.JobsProvider;
+import org.springframework.stereotype.Component;
+
+import java.util.List;
+
+@Component
+public class DefaultGetAllJobsOfLastDayUseCase implements GetAllJobsOfLastDayUseCase {
+    private final JobsProvider jobsProvider;
+    private final GetAllWorkflowRunsOfLastDayUseCase getAllWorkflowRunsOfLastDayUseCase;
```

Comment:
I'm not sure about this, but dependencies between usecases doens't seem like a good idea.